framework-test/src/main/java/org/checkerframework/framework/test/TypecheckResult.java

```diff
@@ -145,10 +145,8 @@ public static TypecheckResult fromCompilationResults(
             CompilationResult result,
             List<TestDiagnostic> expectedDiagnostics) {
 
-        boolean usingAnomsgtxt = configuration.getOptions().containsKey("-Anomsgtext");
         final Set<TestDiagnostic> actualDiagnostics =
-                TestDiagnosticUtils.fromJavaxDiagnosticList(
-                        result.getDiagnostics(), usingAnomsgtxt);
+                TestDiagnosticUtils.fromJavaxDiagnosticList(result.getDiagnostics(), true);
```

Comment:
How about adding a comment here and on line 101 in `TypecheckExecutor` to make the connection explicit?
So in case we want to support tests that don't use `-Anomsgtext` in the future, we'll find the two places more easily?